syfertext/token.py

```diff
@@ -2,12 +2,28 @@
 
 import syft as sy
 import torch
+from syft.generic.string import String
+
+from syft.generic.object import AbstractObject
+from syft.workers.base import BaseWorker
+
+from typing import List
+from typing import Union
 
 hook = sy.TorchHook(torch)
 
 
-class Token:
-    def __init__(self, doc, token_meta: "TokenMeta"):
+class Token(AbstractObject):
+    def __init__(
+        self,
+        doc: "Doc",
+        token_meta: "TokenMeta",
+        id: int = None,
+        owner: BaseWorker = None,
+        tags: List[str] = None,
+        description: str = None,
+    ):
```

Comment:
please remove `tags` and `description`

Reply:
Might be used for Entity Recognizer.

Reply:
Why would you need tags and description. Those are use for PyGrid search